### DISCUSSION TITLE
Show progressbar for mdns search progress

### DISCUSSION THREAD

**Discussion by TheBlubb14** [susch19/Smarthome#11] [Ideas]
E.g. the timer for mdns is 30 seconds, then have a progressbar counting up to 30 seconds max, so the user sees the progress till we show a server. Also maybe lower the timeout

**Comment by susch19** [comment]
Maybe we can even rework the mdns to not need 30 seconds? Because in local network you can broadcast more often and maybe with the providers from riverpod we can do this smarter
